python/tvm/topi/cuda/reduction.py

```diff
@@ -114,6 +114,7 @@ def schedule_reduce(outs):
     outs = [outs] if isinstance(outs, te.tensor.Tensor) else outs
     sch = te.create_schedule([x.op for x in outs])
     scheduled_ops = []
+    enable_auto_inline = tvm.te.schedule.EnableAutoInline(sch)
```

Comment:
Will this too aggressive for some other workloads? Since `EnableAutoInline` returns true as long as there's an unscheduled internal stage with reduce axis, I'm worrying whether it may be too aggressive if a workload has many stages or has a large stage. In this case, the CUDA kernel launching may fail in run time due to out of resource.

cc @vinx13 @icemelon @Hzfengsy @junrushao1994

Reply:
So I made it not too aggressive. By the way, **EnableAutoInline** only returns true when all no-output stages are unscheduled and their reduce_axis have zero size.

Reply:
Ah yeah. I missed that would be evaluated only once instead of per stage. It should be fine then.